hivemind/proto/p2pd.proto

```diff
@@ -160,3 +160,7 @@ message PSResponse {
   repeated string topics = 1;
   repeated bytes peerIDs = 2;
 }
+
+message P2PRPCError {
```

Comment:
nit: I suggest to rename it to just `RPCError` since everything in this file is related to p2p :)